from_cpython/CMakeLists.txt

```diff
@@ -158,21 +156,7 @@ add_library(FROM_CPYTHON OBJECT ${STDMODULE_SRCS} ${STDOBJECT_SRCS} ${STDPYTHON_
 add_dependencies(FROM_CPYTHON copy_stdlib)
 
 set(STDMODULES
-    ${CMAKE_BINARY_DIR}/lib/python2.7/lib-dynload/bz2.pyston.so
```

Comment:
ok, I am fine with keep these definitions. And what about comment out bunch of codes inside setup.py. My idea is enable those commented code when necessary. For example, this PR comment out the `tk` related code. And we can enable it in my `tk_support` PR. But not enable it for now. What do you think?